components/assetcard/package.json

```diff
@@ -53,5 +53,6 @@
 	],
 	"publishConfig": {
 		"access": "public"
-	}
+	},
+	"componentBetaName": "asset-card-beta"
```

Comment:
On the docs site, the [asset card guidelines link](https://opensource.adobe.com/spectrum-css/assetcard.html) actually goes to the [card guidelines](https://spectrum.adobe.com/page/cards/). 

There is an [asset card beta on spectrum contributions](https://spectrum-contributions.corp.adobe.com/page/asset-card-beta/). That is what is used here instead. Any reservations against that?